library/src/main/java/com/okta/appauth/android/OAuthClientConfiguration.java

```diff
@@ -194,13 +194,12 @@ void readConfiguration(@NonNull final JSONObject jsonObject)
             throws InvalidJsonDocumentException {
         JsonParser jsonParser = JsonParser.forJson(jsonObject);
 
-        //We can not take hash code directly from JSONObject
-        //because JSONObject does not follow java has code contract
-        mConfigHash = jsonObject.toString().hashCode();
 
         mClientId = jsonParser.getRequiredString("client_id");
         mRedirectUri = jsonParser.getRequiredUri("redirect_uri");
         mEndSessionRedirectUri = jsonParser.getRequiredUri("end_session_redirect_uri");
+        mDiscoveryUri = jsonParser.getRequiredHttpsUri("issuer_uri")
```

Comment:
We need to think about changing this json structor before 1.0 (not on this PR though)

Reply:
It doesn't match our other java libraries. The json path should likely be `okta.oauth2.*` or `okta.client.*`

Reply:
@bdemers Let's discuss those changes for future pull request. As far as I can see we store our current **configuration json** in **raw** folder.